systemverilog-plugin/UhdmAst.cc

```diff
@@ -632,6 +699,7 @@ static AST::AstNode *expand_dot(const AST::AstNode *current_struct, const AST::A
             // There should be only 1 AST_DOT node children
             log_assert(!sub_dot);
             sub_dot = expand_dot(current_struct_elem, c);
+            break;
```

Comment:
I think we don't want this, multiple ``AST_DOTs`` doesn't make sense, but I think we should check for this.

Beside that, I think after ``AST_DOT`` there can be ``AST_RANGE`` and with this break, we won't find it.

Reply:
Now I realise there's the `log_assert(!sub_dot)` so the valid pointer won't be overwritten. It looks like the `break` is unneeded, so I've removed it and run the CI again.